ui/tests/integration/components/certificate-card-test.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'vault/tests/helpers';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+const SELECTORS = {
+  label: '[data-test-certificate-label]',
+  value: '[data-test-certificate-value]',
+};
+
+module('Integration | Component | certificate-card', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders without a CA Certificate', async function (assert) {
+    await render(hbs`<CertificateCard />`);
+
+    assert.dom(SELECTORS.label).hasText('PEM Format', 'The label text is correct');
+    assert.dom(SELECTORS.value).hasNoText('The is no value for the CA Certificate');
+  });
+
+  test('it renders with a small CA Certificate', async function (assert) {
+    await render(hbs`<CertificateCard @kubernetesCaCert="test"/>`);
```

Comment:
If we update the argument name, we would also need to update the tests to pass the correct names :)